tools/fleetctl-npm/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fleetctl",
-  "version": "v4.2.0",
+  "version": "v4.2.0-2",
```

Comment:
Why -2?

Reply:
NPM versions are immutable, so this is the only way to fix a bad release. Took 3 tries to get it right ;)

Reply:
Ah, TIL